imu_topic: "/imu/data"
left_event_topic: "/prophesee/left/events"
right_event_topic: "/prophesee/right/events"
left_regular_topic: "/stereo/left/image_mono"
right_regular_topic: "/stereo/right/image_mono"
rgbd_color_topic: "/kinect/rgb_image"
rgbd_depth_topic: "/kinect/depth_image"
lidar_topic: "/ouster/point_cloud"
gt_topic: "/gt/pose"
